app/javascript/packages/phone-input/index.ts

```diff
@@ -148,32 +142,29 @@ export class PhoneInputElement extends HTMLElement {
       localizedCountries: countryCodePairs,
       onlyCountries: supportedCountryCodes,
       autoPlaceholder: 'off',
-      allowDropdown: this.hasDropdown,
```

Comment:
For reference, the default is `true`:

https://github.com/jackocnr/intl-tel-input/blob/511870c0151d46c9bcbaf0b4b9cdd88a6fbd4096/src/js/intl-tel-input.ts#L85